R/ExperimentColorMap.R

```diff
@@ -257,11 +257,8 @@ synchronizeAssays <- function(ecm, se){
         ecm_warning)
     }
     # Fetch named-matched assay colormaps
-    new_ecm_assays <- sapply(
-      se_assay_names,
-      function(x){assayColorMap(ecm, x)}
-    )
-  
+    new_ecm_assays <- lapply(se_assay_names, function(x){assayColorMap(ecm, x)})
```

Comment:
Or maybe just `lapply(se_assay_names, assayColorMap, ecm=ecm)` (or whatever the first arg is called).